tests/conftest.py

```diff
@@ -217,31 +215,67 @@ def _framework_fixture_inner(request, app, slug, **kwargs):
     "slug": "g-cloud-8",
     "framework": "g-cloud",
     "framework_agreement_details": _generic_framework_agreement_details,
+    "applications_close_at_utc": "2000-01-01T00:00:00.000000Z",
+    "intention_to_award_at_utc": "2000-01-01T00:00:00.000000Z",
+    "clarifications_close_at_utc": "2000-01-01T00:00:00.000000Z",
+    "clarifications_publish_at_utc": "2000-01-01T00:00:00.000000Z",
+    "framework_live_at_utc": "2000-01-01T00:00:00.000000Z",
+    "framework_expires_at_utc": "2000-01-01T00:00:00.000000Z",
```

Comment:
Nice idea! Much better than a conditional migration 😸 I'll try running this locally and see how it behaves.

Reply:
Thanks! 👍 Yeah it seems to work for me. Let me know. :)

It doesn't, of course, get rid of our problem of relying on migration data in tests. But that feels like a separate (and bigger) issue/

Reply:
Oops, I read this as the wrong thing.

We are actually still doing a conditional migration in 1150. Any framework that doesn't have dates set will get them assigned.

This stuff in conftest didn't work and I should probably remove it.

Reply:
Oops, I read this as the wrong thing.

We are actually still doing a conditional migration in 1150. Any framework that doesn't have dates set will get them assigned.

This stuff in conftest didn't work and I should probably remove it.

Reply:
Except I won't remove it as it needs to be there if for frameworks getting created. So it's fine, but not the solution to our original problem :)